sections/semantics-textlevel.include

```diff
@@ -2957,11 +2957,28 @@
   value <a attr-value for="global/dir"><code>ltr</code></a> to specify a left-to-right override and with the value <a attr-value for="global/dir"><code>rtl</code></a> to
   specify a right-to-left override. The <a attr-value for="global/dir"><code>auto</code></a> value must not be specified.
 
-
   <p class="note">
     This element <a href="#bidireq">has rendering requirements involving the bidirectional algorithm</a>.
   </p>
 
+  <div class="example">
+      This example shows how the <{bdo}> element can be used to alter directionality formatting:
+
+      <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+
+      This effect is achieved by applying the <code>rtl</code> attribute to the <{bdo}> element:
+
+      <xmp highlight="html">
+        <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+      </xmp>
+  </div>
+
+  <div class="example">
+      If character encoding isn't appropriately defined, the characters of <abbr>RTL</abbr> languages
+      may appear in reverse order. For example:
+
+      <bdi><bdo dir="ltr">עֶזרָה</bdo></bdi>
```

Comment:
Can you add in some `<xmp>` example code so that readers can see how this was achieved?

Reply:
Specifically because people who can't read Hebrew won't be able to tell what's wrong.

Reply:
Actually, can you explicitly say what is going on with the example? E.g. hebrew characters normally read right to left, so ...

(You might want to use `ruby` on the characters, or something that indicates what happened).

Reply:
I was under the impression that `ruby` was reserved for kanji?  Otherwise, does

>In this example, Hebrew characters have been reversed from their normal right to left order.

sound like an acceptable explanation for the `xmp` example code?

Reply:
(fwiw, actually ruby doesn't support all you need for a general annotation mechanism, and is developed with the particular needs of CJKM in mind, so i don't usually recommend using it for general cases unless one proceeds with caution, and one avoids a suggesting to uninformed readers  that ruby is an adequate general solution.)